core/src/main/java/org/elasticsearch/cluster/health/ClusterIndexHealth.java

```diff
@@ -37,12 +37,9 @@
 import java.util.Locale;
 import java.util.Map;
 
-import static org.elasticsearch.action.admin.cluster.health.ClusterShardHealth.readClusterShardHealth;
+import static org.elasticsearch.cluster.health.ClusterShardHealth.readClusterShardHealth;
 
-/**
- *
- */
-public class ClusterIndexHealth implements Iterable<ClusterShardHealth>, Streamable, ToXContent {
+public final class ClusterIndexHealth implements Iterable<ClusterShardHealth>, Streamable, ToXContent {
```

Comment:
Because this class is not intended to be inherited from and I want to enforce that. I like implementations to be as restrictive as possible. :)